spec/fabricators/web_push_subscription_fabricator.rb

```diff
@@ -8,4 +8,6 @@
     Base64.urlsafe_encode64(ecdh_key)
   end
   key_auth { Base64.urlsafe_encode64(Random.new.bytes(16)) }
+  user { Fabricate(:user) }
+  access_token { |attrs| Fabricate.build(:access_token, resource_owner_id: attrs[:user].id) }
```

Comment:
```suggestion
  access_token { |attrs| Fabricate.build(:accessible_access_token, resource_owner_id: attrs[:user].id) }
```

Otherwise we might get some random failures with expired access tokens.